packages/components/src/ProgressBar/ProgressBar.tsx

```diff
@@ -12,12 +13,24 @@ interface ProgressBarProps {
    * The total steps to use. For percentages you can set this to 100.
    */
   readonly totalSteps: number;
+
+  /**
+   * Changes the size to small
```

Comment:
Suggestion:
```suggestion
   * Set the size of the progress bar (optional)
```

Reply:
`(optional)` will be inferred by the docs viewer.